diff --git a/config.def.h b/config.def.h
index 823e79f..e315c26 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "cherry:pixelsize=13:antialias=false:autohint=false:scalable=false";
 static int borderpx = 2;
 
 /*
